### ISSUE TITLE
BOM needs minor update (8 m3x30mm dowl pens not 4)

### ISSUE THREAD

**Issue by jhandel** [ankurv2k6/daksh-toolchanger-v2#13]
While building a tool holder I noticed that it looked like we need 4 30mm m3 dowls for the axis connector and then 2 for each of the sliders (total 8).  The BOM shows only needing 4.. I double checked your construction video and it also shows needing a total of 8. 

Not a big deal but figured it was worth sharing.

Thanks

**Comment by 3dfiyMyLife** [comment]
Yup, just checked and you are right. Thank you for the discovery. Will try having them updated. There's a few changes as well to the toolheads as we're more towards the new V2ool toolheads now
